components/plugin-manager/implementation/src/main/kotlin/Utils.kt

```diff
@@ -40,3 +40,10 @@ internal fun getInstalledPlugins(pluginType: PluginType): List<PluginDescriptor>
         PluginType.REPORTER -> ReporterFactory.ALL.values.map { it.descriptor }
         PluginType.SCANNER -> ScannerWrapperFactory.ALL.values.map { it.descriptor }
     }
+
+/**
+ * Returns the normalized plugin ID for the given [pluginId] that exactly equals the case-sensitive ID of the plugin
+ * descriptor. If no match is found, `null` is returned.
+ */
+internal fun normalizePluginId(pluginType: PluginType, pluginId: String): String? =
```

Comment:
Hmm. I guess the need for this is good input for https://github.com/oss-review-toolkit/ort/pull/10190, and we should adjust that PR so that this is not needed, or?

Reply:
The function would still be needed to ensure that entries in the database always use the same spelling.